File: Placements.DataAccess/placement/Models/JobpostingsEligiblestudent.cs
using System;
using System.Collections.Generic;

namespace Placements.DataAccess.Placement.Models;

public partial class JobpostingsEligiblestudent
{
    public long Id { get; set; }

    public long? StudentId { get; set; }

    public long? JobPostingId { get; set; }

    public int? StatusId { get; set; }

    public virtual Jobposting? JobPosting { get; set; }

    public virtual Jobstudentstau? Status { get; set; }

    public virtual Tblstudent? Student { get; set; }
}

File: web-app/src/app/services/types/JobpostingsEligibleStudent.ts
import { Jobposting } from "./Jobposting";
import { Jobstudentstatus } from "./Jobstudentstatus";
import { Tblstudent } from "./Tblstudent";

export type JobpostingsEligiblestudent = {
  Id: number;
  StudentId?: number;
  JobPostingId?: number;
  StatusId?: number;
  JobPosting: Jobposting;
  Status?: Jobstudentstatus;
  Student?: Tblstudent;
};
